src/test-harness/SparkLiquidityLayerTests.sol

```diff
@@ -346,50 +565,48 @@ abstract contract SparkLiquidityLayerTests is SpellRunner {
 
         SparkLiquidityLayerContext memory ctx = _getSparkLiquidityLayerContext();
 
-        // NOTE: Using param here because during an upgrade the _sparkLiquidityLayerContext
-        //       will return a controller that is out of date.
-        ForeignController domainController = ForeignController(foreignController);
-
         address domainAlmProxy = address(ctx.proxy);
 
         uint256 domainUsdcProxyBalance = domainUsdc.balanceOf(domainAlmProxy);
         uint256 domainUsdcPsmBalance   = domainUsdc.balanceOf(domainPsm3);
 
         assertEq(domainUsdc.balanceOf(domainAlmProxy), domainUsdcProxyBalance);
 
-        _relayMessageOverBridges();
+        //_relayMessageOverBridges();
```

Comment:
Rm comment

Reply:
This is to fix the issue with the fork/storage. I added a FIXME comment.